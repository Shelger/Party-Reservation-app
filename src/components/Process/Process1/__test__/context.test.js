import React from "react";
import Context from "../context";
import TestRenderer from 'react-test-renderer'; 
import {cleanup} from "@testing-library/react";

afterEach(cleanup);

it("match ProcessDate", () => {
    const context = TestRenderer.create(<Context title = "aloha" regular_price = {1} num_player = {1} additional_price = {1} limit_player = {1} description = "aloha" />).toJSON();
    expect(context).toMatchSnapshot();
})