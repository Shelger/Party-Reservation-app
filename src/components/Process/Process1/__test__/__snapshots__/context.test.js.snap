// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match ProcessDate 1`] = `
<div
  className="MuiBox-root css-1wbqklf"
>
  <p
    className="MuiTypography-root MuiTypography-body1 css-9jxcda-MuiTypography-root"
  >
    aloha
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    $
    1
     up to 
    1
     
    <br />
    $
    1
     per additional player 
    <br />
    Max 
    1
     players 
    <br />
    Party for 
    1
     players 
    <br />
    Two (2) non-private Laser Tag games 
    <br />
    Cheese pizza for all players (toppings +$2) 
    <br />
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  >
    aloha
  </p>
  <form
    autoComplete="off"
    className="MuiBox-root css-1akph9c"
    noValidate={true}
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-alignJustify css-1oetjkv-MuiTypography-root"
    >
      How many people will be attending the event?
    </p>
    <div
      className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
    >
      <label
        className="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary MuiFormLabel-filled css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
        data-shrink={true}
        htmlFor="outlined-number"
        id="outlined-number-label"
      >
        Players
      </label>
      <div
        className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
          defaultValue={10}
          disabled={false}
          id="outlined-number"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="number"
        />
        <fieldset
          aria-hidden={true}
          className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            className="css-1in441m"
          >
            <span>
              Players
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <br />
    <a
      className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
      data-testid="nb"
      href="process/date"
      onBlur={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
    >
      NEXT
      <span
        className="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="ArrowForwardIosIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M6.23 20.23 8 22l10-10L8 2 6.23 3.77 14.46 12z"
          />
        </svg>
      </span>
    </a>
  </form>
</div>
`;
